staramr/blast/pointfinder/PointfinderDatabaseInfo.py

```diff
@@ -202,3 +202,20 @@ def get_resistance_nucleotides(self, gene, nucleotide_mutations):
                 resistance_mutations.append(nucleotide_mutation)
 
         return resistance_mutations
+
+    def get_notes(self, gene, mutation):
+        """
+        Gets the note associated with a particular mutation from the Pointfinder Database table.
+
+        :param gene: The gene.
+        :param mutation: The mutation.
+        :return: A string containtain the Note, if it exists.
+        """
+
+        matches = self._get_resistance_codon_match(gene, mutation)
+        matches = matches.fillna("-")
```

Comment:
Why do you need to fill NA with "-" here? Wouldn't this mean the notes now have a bunch of strings with "-" in them if it was NA? Would it make more sense to have it blank, or leave as NA?

Reply:
This ends up changing the note returned (which is displayed in the `pointfinder.tsv` file). I chose `-` over a space, because I wasn't sure how it would be handled in a TSV file (if consecutive whitespace - tab space tab - would be handled inaccurately). I can change it to an empty string though.

I think leaving it as NA will cause a crash when it tries to write it later, but I can probably write some checks to handle it.

Reply:
Okay, that makes sense.

Is NA where there is no notes? Stripping those entries out might make more sense than keeping them in?

Reply:
I've slept on it and I think I'm going to replace `-` with an empty string. So we would end up with [TAB][TAB] in the event of no notes in the output. Currently, this probably won't make any difference as `Notes` is the last column, but it may in the future.

Yeah, the NA is when there are no notes. With respect to removing those entries, we need to have something to print when the `pointfinder.tsv` table is printed, so if we remove the NAs (or leave them as NAs and return NA), then we'll just be handling the same problem later when we need to print them. They exist in the `PointfinderDatabaseInfo` table, and the getter is just looking up what the corresponding note entry is for a particular mutation.

The getter needs to return something: empty string, filler, None, etc. I tried implementing a solution where if all entries were NA, then the getter would return `None`, and then it would be handled in turn by the caller, but the added complexity of handling all the cases (note exists, no notes exist, multiple notes exist and both are NA, multiple notes exist and some are NA and some are not, and multiple notes exist and there are no NAs) didn't seem to justify removing the more elegant solution of `.fillna("")` and `";".join(...)`.

Reply:
Okay. Empty string makes sense to me. I believe `[TAB][TAB]` is how NA is represented in a TSV format as well (it's similar for CSV, with `[COMMA][COMMA]`, except we replace command with tab.